lisa/node.py

```diff
@@ -501,6 +502,11 @@ def get_information(self) -> Dict[str, str]:
 
         return final_information
 
+    def exec_check_dmesg_oops(self) -> bool:
+        return plugin_manager.hook.check_dmesg_oops(
```

Comment:
1. Group errors from different plugins, if it returns non-empty.
2. Raise failure exception and let the upper layer to deal with it.